extensions/wikia/AdEngine/js/lookup/adapter/appnexusPlacements.js

```diff
@@ -0,0 +1,32 @@
+define('ext.wikia.adEngine.lookup.adapter.appnexusPlacements', [
+	'ext.wikia.adEngine.utils.adLogicZoneParams',
+	'wikia.window'
+], function (zoneParams, win) {
+	'use strict';
+
+	var placements = {
+		mercury: {
+			entertainment: isDevEnviroment() ? '9412980' : '9412992',
```

Comment:
IMO it's better to name things like that (even if it's used once), because if it's named you don't have to think about "what is this", and why it's 123, not 321, it's f.e. SPECIAL_ARTICLE_ID and that's all what we need no know. Value shouldn't be important it's only "configuration thing"